Sources/StreamVideoSwiftUI/Utils/PictureInPicture/StreamAVPictureInPictureVideoCallViewController.swift

```diff
@@ -0,0 +1,50 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import AVKit
+import Foundation
+import StreamVideo
+import StreamWebRTC
+
+@available(iOS 15.0, *)
+final class StreamAVPictureInPictureVideoCallViewController: AVPictureInPictureVideoCallViewController {
```

Comment:
The name is a bit too long, maybe we can do `StreamPiPCallViewController` or something?

Reply:
I don't like the PiP abbreviation and in any ways this is an internal object.